go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/hectane/go-acl v0.0.0-20190604041725-da78bae5fc95
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/madlambda/spells v0.4.2
-	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
+	github.com/pkg/browser v0.0.0-20240102092130-5ac0b6a4141c
```

Comment:
FYI: this library has no releases ... 🤦 the author is trustworthy though (Go maintainer) ...
Imported by 1336 public projects: https://pkg.go.dev/github.com/pkg/browser?tab=importedby
@snakster @mariux